tests/chainer_tests/functions_tests/connection_tests/test_n_step_gru.py

```diff
@@ -95,34 +92,27 @@ def check_forward(self, h_data, xs_data, ws_data, bs_data):
         testing.assert_allclose(hy.data, e_hy, rtol=1e-4, atol=1e-4)
 
     def test_forward_cpu(self):
-        with chainer.using_config('use_cudnn', self.use_cudnn), \
-                chainer.using_config('enable_backprop', True):
-            self.check_forward(self.hx, self.xs, self.ws, self.bs)
-
-    def test_forward_cpu_volatile(self):
-        with chainer.using_config('use_cudnn', self.use_cudnn), \
-                chainer.using_config('enable_backprop', False):
-            self.check_forward(self.hx, self.xs, self.ws, self.bs)
+        self.check_forward(self.hx, self.xs, self.ws, self.bs)
 
-    @attr.gpu
-    def test_forward_gpu(self):
-        with chainer.using_config('use_cudnn', self.use_cudnn), \
-                chainer.using_config('enable_backprop', True):
```

Comment:
I think this test is not required, but instead we need to test 'train' and 'test' mode.